azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -196,11 +208,16 @@ internal class CallingMiddlewareActionHandlerImpl(
                         FatalError(error, ErrorCode.CAMERA_INIT_FAILED)
                     )
                 )
+            } else {
```

Comment:
this is for skipSetup flow. As call start needs to be trigerred automatically rather than with some user tap event. So we should call it right after setup is finished.

Reply:
Kept and updated as discussed.